### DISCUSSION TITLE
Error: ENOENT: no such file or directory, open '/vercel/path0/.next/server/pages/404.html' during Vercel build

### DISCUSSION THREAD

**Discussion by NSpehler** [vercel/community#3385] [Help] [answered]
### Page to Investigate

[REMOVED]

### Steps to Reproduce

Hello,

For two of my Next.js projects, I have this new error since this morning: 
Error: ENOENT: no such file or directory, open '/vercel/path0/.next/server/pages/404.html'

Here are the 2 separate deployement URLs:
[REMOVED]

This is happening after the build is seemingly successful on Vercel. When running the build command locally, there's no issue.

On one of these projects, there was no change at all in the codebase since this error started appearing. The only workaround I have found was to delete the custom pages/404.js from the codebase altogether, which fixes the issue.

Thank you for your help,
Nicolas

**Comment by swarnava** [comment, marked as answer]
Hi,

We have pushed a fix for this few minutes ago. Could you redeploy and let us know how it goes?

**Reply by NSpehler** [reply]
It's working thank you! 🙌
